components/governance/org.wso2.carbon.governance.taxonomy/src/main/java/org/wso2/carbon/governance/taxonomy/beans/PaginationBean.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.governance.taxonomy.beans;
+
+/**
+ * This bean class will store taxonomy pagination meta data
+ */
+public class PaginationBean {
+
+    private int startNode;
+    private int endNode;
+
+    public int getEndNode() {
+        return endNode;
+    }
+
+    public void setEndNode(int endNode) {
+        this.endNode = endNode;
+    }
+
+    public int getStartNode() {
+        return startNode;
+    }
+
+    public void setStartNode(int startNode) {
+        this.startNode = startNode;
+    }
+
```

Comment:
Thank you Hasitha for the advice. I have added method comments for all getters and setters.
this fixed with https://github.com/wso2/carbon-governance/pull/256/commits/677bdff1879f1d46104c09434267cecf2a92d086